src/list.rs

```diff
@@ -107,3 +108,29 @@ pub fn choose_password_in_list(
     println_stderr!("");
     request_password_index_from_stdin(passwords, prompt)
 }
+
+pub fn search_and_choose_password<'a>(
+    store: &'a PasswordStore,
+    query: &str,
+    with_numbers: bool,
+    prompt: &str,
+) -> Option<&'a Password> {
+    let passwords = store.search_passwords(query);
+    if passwords.len() == 0 {
+        println_stderr!("I can't find any passwords for \"{}\"", query);
+        return None;
+    }
+
+    if let Some(&password) = passwords.iter().find(|p| {
+        p.name.to_lowercase() == query.to_lowercase()
+    }) {
+        return Some(&password)
+    }
+
+    if passwords.len() == 1{
```

Comment:
This seems to make things simpler, but could lead to unwanted disclosure of a password on the wrong site.

Imagine you think you have your Discord.gg password saved in Rooster, but you actually only have your Google password saved in Rooster.

You might go on http://discord.gg, then retrieve your Discord password with `rooster get gg` and paste it in http://discord.gg/.

Woops, you've just disclosed your Google password to Discord.

Obviously, Discord and Google are reputable, but if you use some less reputable sites (as I do, yes, I do), well then you don't want to inadvertently disclose a password like that.

Got it ? So what this means is that if the `passwords.len() == 1`, we need to ask the user if the password is the right one.